src/deluge/io/midi/midi_device_manager.cpp

```diff
@@ -61,6 +61,7 @@ MIDIDeviceUSBUpstream upstreamUSBMIDIDevice_port1{0};
 MIDIDeviceUSBUpstream upstreamUSBMIDIDevice_port2{1};
 MIDIDeviceUSBUpstream upstreamUSBMIDIDevice_port3{2};
 MIDIDeviceDINPorts dinMIDIPorts{};
+MIDIDeviceLoopback loopbackMidi{};
```

Comment:
Add to config menu to allow MPE settings to be made - https://github.com/SynthstromAudible/DelugeFirmware/blob/4f9dfcf9f5b44c95fb412da716dd0ce65ba18630/src/deluge/gui/menu_item/midi/devices.cpp#L112

Reply:
Loopback present in midi devices menu now